tasks/main.yml

```diff
@@ -4,3 +4,7 @@
 
 - include: RedHat.yml
   when: ansible_os_family == 'RedHat'
+
+- name: install cleanup cron
```

Comment:
Can you make this optional? Not everyone would want to clean up. Also, can you add comment to explain what it's doing? Not sure what exactly gets deleted and I'm a bit scared of it :)